### PR TITLE
feat(blade): rebranding whitelabel fix

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: packages/blade/src/components/Badge/Badge.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import type { StyledPropsBlade } from '~components/Box/styledProps';
 import type { StringChildrenType, TestID } from '~utils/types';
 import { getStringFromReactText } from '~src/utils/getStringChildren';
 import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
-import { isReactNative, makeSize } from '~utils';
+import { isReactNative, makeSize, useTheme } from '~utils';
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Badge/Badge.tsx:16]: why

File: packages/blade/src/storybook-recipes/ThemePlayground/ThemeSelector.tsx [hunk 2]
```diff
@@ -68,7 +72,7 @@ const ColorSelector = ({
 }): React.ReactElement => {
   return (
     <ColorSelection color={color} onClick={onClick} isSelected={isSelected}>
-      {isSelected ? <CheckIcon size="large" color="action.icon.primary.default" /> : null}
+      {isSelected ? <CheckIcon size="large" color="interactive.icon.staticWhite.normal" /> : null}
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/storybook-recipes/ThemePlayground/ThemeSelector.tsx:75]: use onPrimary here ?

Comment (anuraghazra) [reply packages/blade/src/storybook-recipes/ThemePlayground/ThemeSelector.tsx:75]: done

File: packages/blade/src/components/Badge/Badge.tsx (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/Badge/Badge.tsx]: removed

**Comment by chaitanyadeorukhkar** [comment]
Lets use the primary color on carousel's quotes like we do on master? Right now it always remains blue
Screenshot 2024-01-12 at 12 58 03 PM

**Comment by chaitanyadeorukhkar** [comment]
Focus ring color is not changing for multiple components like Checkbox, Radio, Accordion, Switch, Chips... can you check this once? I created a common `getFocusRingStyles` function that uses `surface.border.primary.muted` which should ideally change as per the white labeled theme
Screenshot 2024-01-12 at 1 00 25 PM

**Comment by chaitanyadeorukhkar** [comment]
Carousel's button's arrow icons should also follow the primary color, right?
Screenshot 2024-01-12 at 1 03 22 PM

**Comment by anuraghazra** [comment]
> Focus ring color

Now that you mention it, I wonder if we should keep the focus ring color static and not tie it with brand color. 

The focus ring is an fundamental accessibility requirement, if brand color becomes very light then focus ring color won't be visible to the user and could cause accessibility issues. 

https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html

**Comment by anuraghazra** [comment]
> Carousel's button's arrow icons should also follow the primary color, right?

Nope, rebranded carousel nav buttons icons are staticBlack.

**Comment by anuraghazra** [comment]
> Lets use the primary color on carousel's quotes like we do on master? Right now it always remains blue

Done.

**Comment by chaitanyadeorukhkar** [comment]
> > Focus ring color
> 
> Now that you mention it, I wonder if we should keep the focus ring color static and not tie it with brand color.
> 
> The focus ring is an fundamental accessibility requirement, if brand color becomes very light then focus ring color won't be visible to the user and could cause accessibility issues.
> 
> https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html

Hmm. That is fair. I think that we can think about on its own but right now we're using primary color token for the focus ring ideally, it should use the primary color when it changes as well?

**Comment by chaitanyadeorukhkar** [comment]
> > Carousel's button's arrow icons should also follow the primary color, right?
> 
> Nope, rebranded carousel nav buttons icons are staticBlack.

Why are they blue in this screenshot then? 🤔 
Screenshot 2024-01-12 at 1 03 22 PM

**Comment by chaitanyadeorukhkar** [comment]
@anuraghazra Tests are failing. Mostly looks good to me. Just need to check

1. Focus ring color
2. Carousel's blue button

**Comment by anuraghazra** [comment]
> Why are they blue in this screenshot then? 🤔

Wierd, are you checking the correct chromati link? Because it's not possible for it to be blue. 

image

**Comment by chaitanyadeorukhkar** [comment]
> > Why are they blue in this screenshot then? 🤔
> 
> Wierd, are you checking the correct chromati link? Because it's not possible for it to be blue.
> 

Might be the wrong link. Checked again. Looks good

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Comment by anuraghazra** [comment]
> Focus ring color

I've standardized all components to use the `getFocusRingColor` function.

**Pull request merged**